x/consensus/types/msgs.go

```diff
@@ -0,0 +1,51 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+// bank message types
+const (
+	TypeMsgUpdateParams = "update_params"
+)
+
+var _ sdk.Msg = &MsgUpdateParams{}
+
+// GetSigners returns the signer addresses that are expected to sign the result
+// of GetSignBytes.
+func (msg MsgUpdateParams) GetSigners() []sdk.AccAddress {
+	authority, _ := sdk.AccAddressFromBech32(msg.Authority)
+	return []sdk.AccAddress{authority}
+}
+
+// GetSignBytes returns the raw bytes for a MsgUpdateParams message that
+// the expected signer needs to sign.
+func (msg MsgUpdateParams) GetSignBytes() []byte {
+	return []byte{}
```

Comment:
We should add amino support here

Reply:
this is to keep consistency with other messages? not sure its needed as gov will execute and/or groups/multisig. I guess for individual accounts if a chain wanted?

Reply:
We should still allow a proposer to do a MsgSubmitProposal using ledger.

Reply:
oh I had it backwards in my head. I thought it worked differently will add it